lte/gateway/python/integ_tests/s1aptests/test_sctp_shutdown_while_mme_is_stopped.py

```diff
@@ -55,9 +55,10 @@ def test_sctp_shutdown_while_mme_is_stopped(self):
         self._s1ap_wrapper._s1_util.receive_emm_info()
 
         print("Stopping MME service")
```

Comment:
Comment needs to be updated as we are stopping multiple services.

Reply:
Adapted.